data/sql_updates/create_candidates_view.sql

```diff
@@ -29,8 +29,7 @@ select
     max(candprops.cand_nm) as name
 from dimcand
     left join (select distinct on (cand_sk) cand_sk, election_yr, cand_status, ici_code from dimcandstatusici order by cand_sk, election_yr desc) csi_recent using (cand_sk)
-    left join dimcandstatusici csi_all using (cand_sk)
-    inner join dimcandoffice co on co.cand_sk = dimcand.cand_sk and (csi_recent.election_yr is null or co.cand_election_yr = csi_recent.election_yr)  -- only joined to get to dimoffice
+    inner join dimcandoffice co on co.cand_sk = dimcand.cand_sk --and (csi_recent.election_yr is null or co.cand_election_yr = csi_recent.election_yr)
```

Comment:
This isn't a new change in this PR, but is it expected to join `dimcand` and `dimcandoffice` with a `LEFT JOIN` in `ofec_candidates_mv` but an `INNER JOIN` in `ofec_candidate_detail_mv`? My possibly naive expectation is that we would use the same joins so that every candidate in one view also shows up in the other.